src/index.js

```diff
@@ -44,4 +44,10 @@ syntheticEvents.forEach(eventName => {
 
 // just re-export everything from dom-testing-library
 export * from 'dom-testing-library'
```

Comment:
I was actually hoping this PR would be made to dom-testing-library. Then we'll get it automatically in react-testing-library because of this.

Reply:
Hi @kentcdodds, I noticed that dom-testing-library already exported getQueriesForElement as something else:

`export {getQueriesForElement as bindElementToQueries}`

Is it okay to still export it as yet another thing? like: 

`export {getQueriesForElement as bindElementToQueries}`
`export {getQueriesForElement as within}`

I've not done such export before :smile:

Reply:
Yup, that works just fine :+1: You can even do it in a single line:

```js
export {getQueriesForElement as bindElementToQueries, getQueriesForElement as within}
```

Reply:
Thanks man!

How about the docs, should I also move it to dom-testing-library?

Reply:
Might be good to have it in both places actually. I think we really need a documentation website to consolidate this, but I don't have time to build it right now 😅

Reply:
Have updated this to add only docs.